packages/contracts/contracts/THUSDToken.sol

```diff
@@ -94,8 +93,6 @@ contract THUSDToken is Ownable, CheckContract, ITHUSDToken {
         _HASHED_VERSION = hashedVersion;
         _CACHED_CHAIN_ID = block.chainid;
         _CACHED_DOMAIN_SEPARATOR = _buildDomainSeparator(_TYPE_HASH, hashedName, hashedVersion);
-        governanceTimeDelay = _governanceTimeDelay;
-        require(governanceTimeDelay <= 30 weeks, "Governance delay is too big");
```

Comment:
I'd keep it in the constructor anyway, otherwise will be period of time when delay is zero

Reply:
Addressed in [refactor: add governance time delay in thusd constructor](https://github.com/Threshold-USD/dev/pull/113/commits/9f771a49a984e3122ea25dd5881305289819e1f9)